buildchain/buildchain/constants.py

```diff
@@ -19,7 +19,7 @@
 # URLs of the main container repositories.
 CALICO_REPOSITORY               : str = 'docker.io/calico'
 COREOS_REPOSITORY               : str = 'quay.io/coreos'
-DEX_REPOSITORY                  : str = 'quay.io/dexidp'
+DEX_REPOSITORY                  : str = 'dexidp'
```

Comment:
Should this be `docker.io/dexidp`? Not fond of `docker.io` being special-cased.

Reply:
Right to be consistent